File: src/clojure/leiningen/new/libgdx_clojure/android-launcher.clj
(ns {{android-namespace}}
  (:require [{{namespace}}]))

(gen-class
  :name {{package}}.{{activity}}
  :extends com.badlogic.gdx.backends.android.AndroidApplication
  :exposes-methods [superOnCreate onCreate])
(defn -onCreate [this saved-instance-state]
  (superOnCreate this saved-instance-state)
  (.initialize this ({{namespace}}.Game.) false))

File: src/clojure/leiningen/new/game_java/AndroidLauncher.java
package {{package}};

import com.badlogic.gdx.backends.android.AndroidApplication;

public class {{android-class-name}} extends AndroidApplication {
	public void onCreate (android.os.Bundle savedInstanceState) {
		super.onCreate(savedInstanceState);
		initialize(new {{class-name}}());
	}
}
